.travis.yml

```diff
@@ -32,7 +32,8 @@ matrix:
     - env: PACKAGE_TYPE=deb RESTY_IMAGE_BASE=debian RESTY_IMAGE_TAG=testing
 
 install:
-  - git clone --single-branch -b next https://github.com/Kong/kong.git kong
+  - git clone https://github.com/Kong/kong.git kong
+  - cd kong && git reset --hard 1fc45e5
```

Comment:
remove once https://github.com/Kong/kong/pull/4279 is merged

Reply:
ahh, missed this note, sorry

Reply:
Kong/kong#4279 is merged so maybe this can be removed now.

Reply:
4b1d474 just waiting on travis-ci now